modules/mauth-test-utils/src/main/java/com/mdsol/mauth/test/utils/ProtocolTestSuiteHelper.java

```diff
@@ -0,0 +1,135 @@
+package com.mdsol.mauth.test.utils;
+
+import com.mdsol.mauth.test.utils.model.AuthenticationHeader;
+import com.mdsol.mauth.test.utils.model.SigningConfig;
+import com.mdsol.mauth.test.utils.model.UnsignedRequest;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.File;
+import java.io.FilenameFilter;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class ProtocolTestSuiteHelper {
+
+  final static String TEST_SUITE_RELATIVE_PATH =
+      System.getenv("TEST_SUITE_RELATIVE_PATH") != null ? System.getenv("TEST_SUITE_RELATIVE_PATH")
+          : "../../../mauth-protocol-test-suite";
+  final static String MWSV2_TEST_CASE_PATH = TEST_SUITE_RELATIVE_PATH + "/protocols/MWSV2/";
```

Comment:
Let's use git submodules for this otherwise your test failures are unreproducible.

Reply:
Yes, once `mauth-protocol-test-suite` goes to public, will utilize git submodule

Reply:
Let's have a follow up story once the suite is public and 🙈 for now.

Reply:
Why does it need to be public for git submodules to work?

Reply:
because this repo is public. it would work _for us_ but for anyone outside it wouldn't (not that there's anyone I guess but who knows?)

Reply:
Oh duh yeah ofc I forgot this repo is public too :sweat_smile:

I'd rather wait for that repo to go public and then make sure this repo's CI test against it. Currently it just succeeds if no test cases are found so I don't think there's much point merge this?